app/src/main/res/layout/chapter_filter_popup.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:background="?attr/primaryBlackBackground">
+
+    <!-- Tab Layout -->
+    <com.google.android.material.tabs.TabLayout
```

Comment:
This style is completely different from any tab in the entire app. Please use the appropriate style